db/organizations/panasonic.eno

```diff
@@ -0,0 +1,9 @@
+name: Panasonic
+website_url: https://www.panasonic.aero/
+privacy_policy_url: https://www.panasonic.aero/privacy-policy/
+privacy_contact: privacy@panasonic.aero
+country: US
+description: Panasonic provides services such as internet which can be used in-flight.
```

Comment:
Since it is ultimately owned by Panasonic, we should use country: JP. As a company URL here, I do not have a strong opinion; could be either https://www.panasonic.com/ or https://holdings.panasonic/global/

(In the pattern .eno, using https://www.panasonic.aero/ is good though.)

Here would be an example for a company description:

```
Panasonic Holdings Corporation is a multinational electronics company based in Kadoma, Osaka, Japan, offering a diverse range of products and services, including consumer electronics, rechargeable batteries, and home renovation. Among its subsidiaries is Panasonic Avionics Corporation, which specializes in providing customized in-flight entertainment and communication devices to airlines worldwide.
```

Reply:
I updated the file to "panasonic avionics" as wikipedia says so. The HQ is located in Japan but I'm not sure what value we should use for the privacy contact. They have one per location.